dist/dfu.js

```diff
@@ -46,33 +46,23 @@
 
     var LITTLE_ENDIAN = true;
     
-    var packetSize = 20;
-    var notifySteps = 40;
+    var PACKET_SIZE = 20;
```

Comment:
"Global" variables like this aren't usually capitalized in JavaScript, that's more a 'C' const thing.